seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcHighGoIT.java

```diff
@@ -0,0 +1,311 @@
+    }
+
+    @Override
+    protected Class<?> loadDriverClass() {
+        return super.loadDriverClassFromUrl();
+    }
+
+    @Override
+    Pair<String[], List<SeaTunnelRow>> initTestData() {
+        List<SeaTunnelRow> rows = new ArrayList<>();
+        for (Integer i = 0; i < GEN_ROWS; i++) {
+            SeaTunnelRow row =
+                    new SeaTunnelRow(
+                            new Object[] {
+                                i,
+                                String.valueOf(i),
+                                String.valueOf(i),
+                                String.valueOf(i),
+                                i % 2 == 0,
+                                i,
+                                i,
+                                Long.valueOf(i),
+                                BigDecimal.valueOf(i * 10.0),
+                                BigDecimal.valueOf(i * 0.01),
+                                Float.parseFloat("1.1"),
+                                Double.parseDouble("1.111"),
+                                i,
+                                Long.valueOf(i),
+                                LocalDate.of(2024, 12, 12).atStartOfDay(),
+                                LocalDateTime.of(2024, 12, 12, 10, 0),
+                                "Testing"
+                            });
+            rows.add(row);
+        }
+
+        return Pair.of(fieldNames, rows);
+    }
+
+    @Override
+    public String quoteIdentifier(String field) {
+        return "\"" + field + "\"";
+    }
+
+    @Override
+    protected void clearTable(String database, String schema, String table) {
+        clearTable(schema, table);
+    }
+
+    @Override
+    protected String buildTableInfoWithSchema(String database, String schema, String table) {
+        return buildTableInfoWithSchema(schema, table);
+    }
+
+    @Override
+    GenericContainer<?> initContainer() {
+        GenericContainer<?> container =
+                new GenericContainer<>(HIGHGO_IMAGE)
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases(HIGHGO_ALIASES)
+                        .withLogConsumer(
+                                new Slf4jLogConsumer(DockerLoggerFactory.getLogger(HIGHGO_IMAGE)));
+        container.setPortBindings(
+                Lists.newArrayList(String.format("%s:%s", HIGHGO_PORT, HIGHGO_PORT)));
+
+        return container;
+    }
+
+    @Override
+    protected void initCatalog() {
+        String jdbcUrl = jdbcCase.getJdbcUrl().replace(HOST, dbServer.getHost());
+        catalog =
+                new HighGoCatalog(
+                        DatabaseIdentifier.HIGHGO,
+                        jdbcCase.getUserName(),
+                        jdbcCase.getPassword(),
+                        JdbcUrlUtil.getUrlInfo(jdbcUrl),
+                        SCHEMA);
+        // set connection
+        ((HighGoCatalog) catalog).setConnection(jdbcUrl, connection);
```

Comment:
I looked at other JDBC test cases that implemented setConnection, such as iris opengauss, So I imitated and implemented it. And if this setconnection is not implemented, it will cause the test code to have a 'No suitable driver found for' issue

Reply:
> why not let `HighGoCatalog` create new jdbc connection by itself?

I looked at other JDBC test cases that implemented setConnection, such as iris opengauss, So I imitated and implemented it. And if this setconnection is not implemented, it will cause the test code to have a 'No suitable driver found for' issue